src/Unused/CLI/SearchResult.hs

```diff
@@ -0,0 +1,62 @@
+module Unused.CLI.SearchResult
+    ( printSearchResults
+    ) where
+
+import Control.Monad (forM_)
+import Text.Printf
+import Unused.Types
+import Unused.CLI.Util
+
+printSearchResults :: TermMatchSet -> IO ()
+printSearchResults termMatchSet =
+    mapM_ (printDirectorySection $ maxWidth) responses
+  where
+    responses = responsesGroupedByPath termMatchSet
+    allSets = listFromMatchSet =<< map snd responses
+    allResults = fmap snd allSets
```

Comment:
you use `map snd` above. I'm assuming `allSets` is a `List` as well so you could use `map` here too for consistency.